server/ServerFSM.cpp

```diff
@@ -116,22 +116,27 @@ namespace {
         }
     }
 
-    std::string GenerateEmpireName(std::list<std::pair<int, PlayerSetupData> >& players) {
+    std::string GenerateEmpireName(const std::string& player_name,
+                                   std::list<std::pair<int, PlayerSetupData> >& players)
+    {
         // load default empire names
         static std::vector<std::string> empire_names = UserStringList("EMPIRE_NAMES");
```

Comment:
will presumably never refill names list, even if some/all are used by players who then leave the game...